### PR TITLE
Styleguide Update

### LINKED ISSUE #9: Button cursor pointer
Avoir un pointeur comme curseur sur le composant bouton. Nécessaire avant de merge #8

### LINKED ISSUE #10: Exécuter une fonction lorsqu'on clique sur un bouton
Une propriété onClick devrait permettre d'exécuter une fonction lorsqu'on clique sur un bouton. Nécessaire avant de merge #8

### LINKED ISSUE #11: Créer un composant Icon
Ce composant doit afficher une icône en SVG. Il serait d'ailleurs judicieux de définir une bibliothèque d'icône pour l'application. Nécessaire avant de merge #8

### PR DESCRIPTION
**Pull request by loichu**
### Button
Available in 2 versions:
* Classic (.btn)
* Danger (.btn--danger)

### Image Container
Contains an image and can be shaped (`Square`, `Circle`, `Rectangle`)  and sized (`Small`, `Medium`, `Large`, `Big`) using enum properties (`Shape` and `Size`. It is responsive out-of-the-box unless you choose to make it `Adaptive` sized in which case it will fill the parent. It is then parent's responsibility to be responsive. The image automatically covers the container in any case.

### Message
Message contained in a box.

### ChatCard
Similar to `Message` excepts it also takes a pseudo which will be displayed on the left of a default image on top of the message. By default the message is a left message (`.chat-card-container`) but it can also be a right message (`chat-card-container--right`).

### Nav
It takes an array of link objects and makes it a list of links.

### UserCard
Exists in 2 versions:
* classic (`main-user-card`)
* full (`main-user-card--full`)

It displays a user with a default profile picture, a pseudo and a rating (`react-stars`). There is also a warning icon on the top right but it doesnt do anything yet nor serve any purpose. In full mode it also displays a location, a rank and a button "Notifier".

### Header
Displays a menu with a default app icon and default user avatar. It takes The user path and the title as well a styleClass as parameters.

### CODE REVIEW

File: components/Atoms/Button/Button.module.scss [hunk 1]
```diff
@@ -1,7 +1,9 @@
-button {
-  padding: 20px;
-}
-
-.red {
-  color: red;
+@import "styles/variables";
+@import "styles/atoms/buttons";
+.btn {
```
Comment (loichu) [inline components/Atoms/Button/Button.module.scss:3]: Should use a pointer cursor

File: components/Atoms/Button/Button.tsx [hunk 2]
```diff
@@ -1,13 +1,17 @@
 import React from 'react';
-import styles from './Button.module.scss';
+import style from "./Button.module.scss";
+
+interface types {
```
Comment (loichu) [inline components/Atoms/Button/Button.tsx:4]: Should take an onClick event

File: components/Molecules/UserCard/UserCard.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+```tsx
+<UserCard 
+    styleClass="main-user-card--full" 
+    pseudo="Lorem Ipsum" 
+    ratingValue={4.5}
+    rank="6H"
+    location="Lausanne"
+>
+<svg
```
Comment (loichu) [inline components/Molecules/UserCard/UserCard.md]: Should make an Icon atom component

File: components/Atoms/Button/Button.module.scss (hunk 1 above)
Comment (loichu) [reply components/Atoms/Button/Button.module.scss:3]: #9

File: components/Atoms/Button/Button.tsx (hunk 2 above)
Comment (loichu) [reply components/Atoms/Button/Button.tsx:4]: #10

File: components/Molecules/UserCard/UserCard.md (hunk 3 above)
Comment (loichu) [reply components/Molecules/UserCard/UserCard.md]: #11

**Review by Deegoh — APPROVED** [review]
LGTM

**Review by MarJC5 — APPROVED** [review]
LGTM

**Pull request merged**